framework/console/ErrorHandler.php

```diff
@@ -65,6 +65,15 @@ protected function renderException($exception)
         } else {
             echo $message . "\n";
         }
+        if (($previous = $exception->getPrevious()) !== null) {
+            $causedMessage = "\n" . $this->formatMessage('Caused by:', [Console::BOLD, Console::FG_GREEN]) . "\n";
+            if (PHP_SAPI === 'cli') {
+                Console::stderr($causedMessage . "\n");
+            } else {
+                echo $causedMessage . "\n";
+            }
+            $this->renderException($previous);
+        }
```

Comment:
@samdark It's probably better to only do this in debug mode, right? If you agree I'll change it.

One thing thogh: I tried to keep "Caused by:" in the same line like the next "Exception 'XyzException' ..." message, but i could not figure out, where the linebreak is added. If you know and agree, that this would look better then please let me know.

Reply:
After more thinking: Chained exceptions should always be rendered, not only in debug mode. They will already look different in debug/non-debug mode. So the logic is fine.

Just not sure about the formatting. Comments welcome.

Reply:
Updated to improve formatting:

![chained_exception](https://user-images.githubusercontent.com/675062/89500946-d1b2ba80-d7c2-11ea-92eb-160f2a295d4f.png)

Reply:
Yes, it's better to do it in debug only. In non-debug mode we aren't printing too many details on purpose.

Reply:
Ok, changed it.